app/admin/tenants/tenants.py

```diff
@@ -207,28 +210,29 @@ async def get_tenants(
             for wallet_record in wallets.results
         ]
 
-    # NOTE: The group id is provided via a plugin so the
-    # controller doesn't have the api for that implement.
-    # Using http call instead via asyncClient
-    governance_client = AsyncClient(
-        base_url=MULTITENANT_AGENT_URL,
-        timeout=120.0,
-        headers={
-            "x-api-key": f"{TENANT_ACAPY_API_KEY}",
-            "content-type": "application/json",
-        },
-        transport=AsyncHTTPTransport(retries=3),
+    # NOTE: Since this is using the groups plugin we need to override the
+    # controller to be aware of this
+    class MultitenancyApi(Consumer):
+        async def get_wallets(self, *, group_id: str) -> WalletListWithGroups:
+            """Query subwallets"""
+            return await self.__get_wallets(
+                group_id=group_id,
+            )
+
+        @returns.json
+        @get("/multitenancy/wallets?group_id={group_id}")
+        def __get_wallets(self, *, group_id: str = None) -> WalletListWithGroups:
+            """Internal uplink method for get_wallets"""
+
+    aries_controller.multitenancy = MultitenancyApi(
+        base_url=aries_controller.base_url, client=aries_controller.client
```

Comment:
Maybe we can do this when we create the aries controller instance?

Reply:
see [this reply](https://github.com/didx-xyz/aries-cloudapi-python/pull/279#discussion_r1031640778)